eng/tox/import_all.py

```diff
@@ -41,7 +41,7 @@ def should_run_import_all(package_name):
 
     # get target package name from target package path
     pkg_dir = os.path.abspath(args.target_package)
-    package_name, namespace, _ = get_package_details(os.path.join(pkg_dir, 'setup.py'))
+    package_name, namespace, _, _ = get_package_details(os.path.join(pkg_dir, 'setup.py'))
```

Comment:
Seems to be missing one return value:
```suggestion
    package_name, namespace, _, _, _ = get_package_details(os.path.join(pkg_dir, 'setup.py'))
```

Reply:
100% agree. Thanks McCoy.